authenticate.js

```diff
@@ -1,25 +1,30 @@
 var path = require('path');
+var fs = require('fs');
 var closureBasePath = path.join(__dirname, '/ext/closure-library/closure/goog' + path.sep);
 var goog = require('closure').Closure({CLOSURE_BASE_PATH: closureBasePath});
 
+var google = require('googleapis');
+var OAuth2 = google.auth.OAuth2;
```

Comment:
not sure if it should be here, feels more appropriate during initialize - instantiate OAuth2 only when needed